deepchecks/vision/README.rst

```diff
@@ -255,7 +256,7 @@ What Do You Need in Order To Start Validating?
 ================================================
 
 - Images (optional: model, predictions and labels)
-- Pytorch environment
+- Relevant environment for running your model (optional) - such as Pytorch, Tensorflow, etc.

```

Comment:
Because we still rely on torch ignite metrics, you still need to install pytorch. We intend to resolve that in the future so we won't depend on either framework.